docker_build.sh

```diff
@@ -52,7 +52,7 @@ if [ -z "$NEO_CLI_CUSTOM_ZIPFN" ]; then
     cp $NEO_CLI_ZIPFN ./neo-cli.zip
 else
     echo "Using custom neo-cli.zip: $NEO_CLI_CUSTOM_ZIPFN"
-    cp $NEO_CLI_CUSTOM_ZIPFN ./neo-cli.zip
+    mv $NEO_CLI_CUSTOM_ZIPFN ./neo-cli.zip
```

Comment:
Leave the cp here please. Makes it easier to reuse downloaded zips

Reply:
Will do. I found it confusing that there were two zips there is all